docker/examples/sensorthings/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM webbben/pygeoapi-sensorthings:latest
```

Comment:
Can you add a `README.md` in `docker/examples/sensorthings` with some high level notes (and pointer to the codebase/setup of `webbben/pygeoapi-sensorthings` ?

Reply:
`webbben/pygeoapi-sensorthings` points to my branch of pygeoapi with the Sensorthings provider. With a merged PR this is redundant. That being said, I will add a readme and think I am going to replace the Dockerfile with a docker-compose file.